.circleci/config.yml

```diff
@@ -66,9 +66,38 @@ jobs:
       - checkout
       - check-changed:
           patterns: op-bindings,op-chain-ops,packages/
+      - restore_cache:
+          name: Restore Yarn Package Cache
+          keys:
+            - yarn-packages-v2-{{ checksum "yarn.lock" }}
       - run:
           name: Install dependencies
-          command: yarn && git diff --exit-code
+          command: yarn install --imutable
```

Comment:
I don't think that `--imutable` does anything. We're still on an old version of [yarn](https://github.com/ethereum-optimism/optimism/blob/35d4a37c2b9a59e9bf16fa4c5e22283f2348c3d2/.yarnrc#L1), and I think ugprading is non-trivial.

Reply:
yarn v2 might be less bad, not sure.